examples/phi2/README.md

```diff
@@ -1,13 +1,35 @@
 # Phi-2
 
+## Install the onnxruntime-genai library
+
+* (Temporary) Build the library according to [the build instructions](../README.md#build-from-source)
+
+* Install the python package
+
+  ```bash
+  cd build/wheel
+  pip install onnxruntime-genai-*.whl
+  ```
+
 ## Get the model
 
+Install the model builder script dependencies
+
 ```bash
-python -m onnxruntime_genai.scripts.export.py -m microsoft/phi-2 -e cpu -p int4 -o model/model.onnx
+pip install numpy
+pip install transformers
+pip install torch
+pip install onnx
+pip install onnxruntime
+```
+
+```bash
+python -m onnxruntime_genai.models.builder -m microsoft/phi-2 -e cpu -p int4 -o model/model.onnx
```

Comment:
I could be mistaken but isn't the script expecting a directory as an input for the -o flag?

Reply:
I hadn't pushed commit to update this

Reply:
Fixed